kartothek/io/testing/read.py

```diff
@@ -539,3 +541,46 @@ def _f(b_c):
         predicates=[[("c", ">=", datetime.datetime(2000, 1, 1))]],
         dates_as_object=True,
     )
+
+
+@pytest.fixture
+def py2_reference_store():
```

Comment:
Same here regarding the py2 compat. Please rephrase this to `binary_column_metadata_store` (or something similar)